main.go

```diff
@@ -39,6 +39,10 @@ func main() {
 		logrus.SetLevel(logrus.DebugLevel)
 	}
 
+	if _, err := os.Stat("/sbin/mkfs.xfs"); os.IsNotExist(err) {
```

Comment:
You should search for the path rather then hard code it.

exec.Lookpath("mkfs.xfs")

Reply:
Fixed.